docs/resources/apm_retention_filter_order.md

```diff
@@ -0,0 +1,24 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_apm_retention_filter_order Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog APM Retention Filters API https://docs.datadoghq.com/api/v2/apm-retention-filters/ resource, which is used to manage Datadog APM retention filters order.
+---
+
+# datadog_apm_retention_filter_order (Resource)
+
+Provides a Datadog [APM Retention Filters API](https://docs.datadoghq.com/api/v2/apm-retention-filters/) resource, which is used to manage Datadog APM retention filters order.
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
+
+- `filter_ids` (List of String) The filter IDs list. The order of filters IDs in this attribute defines the overall APM retention filters order.. If `filter_ids` is empty or not specified, it will import the actual order, and create the resource. Otherwise, it will try to update the order.
```

Comment:
```suggestion
- `filter_ids` (List of String) The filter IDs list. The order of filters IDs in this attribute defines the overall APM retention filters order. If `filter_ids` is empty or not specified, it will import the actual order, and create the resource. Otherwise, it will try to update the order.
```

Reply:
fixed in 9e7bc2855e3d2cd0b6e0391cf661a292c554b530